### ISSUE TITLE
std::bad_alloc error or segmentation fault for some inputs

### ISSUE THREAD

**Issue by kamalsaleh** [gap-packages/NormalizInterface#61]
Example:
```
LoadPackage( "NormalizInterface" );
L := [ [ -1, -1, 0, 0, 0, 0 ], [ -1, 0, 0, 0, 0, 0 ], [ 0, 0, -1, -1, -1, 0 ], 
         [ 0, 0, -1, 0, 0, 0 ], [ 0, 0, 0, -1, 0, 0 ],  [ 1, 1, 0, 0, 0, 0 ], 
         [ 1, 0, 0, 0, 0, 0 ], [ 0, 0, 1, 1, 1, 0 ], [ 0, 0, 1, 0, 0, 0 ], 
         [ 0, 0, 0, 1, 0, 0 ] ];;
C := NmzCone( "inhom_inequalities", L );
# <a Normaliz cone>
NmzModuleGenerators( C );
```
returns either

```
Error, std::bad_alloc in
  result := _NmzConeProperty( cone, prop ); .....

```
or
`Segmentation fault`

**Comment by fingolfin** [comment]
I can reproduce this with Normaliz 3.4.0 and 3.5.1 but not with 3.7.2, so I am tempted to say this is a bug in Normaliz which has since been fixed. I have a version of NormalizInterface using 3.7.2, but this results in a lot of other tests failing (mostly in harmless ways), so it'll be some time until this materializes.

**Comment by fingolfin** [comment]
The crash is also gone when using Normaliz 3.6.3, which is the version shipped with Ubuntu Bionic (18.10) and later.

**Comment by fingolfin** [comment]
It is also gone in Normaliz 3.5.4

**Comment by fingolfin** [comment]
Fixed in NormalizInterface 1.1.0, which comes with Normaliz 3.5.4
